### ISSUE TITLE
Info pop-ups for calculator

### ISSUE THREAD

**Issue by Scottan** [GreenAlgorithms/green-algorithms-tool#36]
Add ? buttons next to some of the questions which create pop-ups explaining what they mean when you hover over. In particular, I wasn't sure what these meant but took a while to dig through to find out:

- Do you know the Power Usage Efficiency (PUE) of your local data centre?
- Do you want to use a Pragmatic Scaling Factor?

**Comment by Llannelongue** [comment]
Yes indeed very good suggestion, this is indeed coming in the release coming out this week.
Closed by #46
